src/components/notification-center/NotificationCenter/index.tsx

```diff
@@ -67,6 +69,8 @@ const NotificationCenter = (): ReactElement => {
 
   const handleClick = (event: MouseEvent<HTMLButtonElement>) => {
     if (!open) {
+      trackEvent(OVERVIEW_EVENTS.NOTIFICATION_CENTER)
```

Comment:
I would put it as a tag on the button. Highly unlikely someone clicks on the bell again to _close_ the popup. It closes on click outside, and most people do that.

Reply:
True, but the `action` is "'Open Notification Center'".